### ISSUE TITLE
Training on ModelNet40

### ISSUE THREAD

**Issue by schakrab** [rusty1s/pytorch_geometric#959]
Hi, 
I was attempting to train on ModelNet40. After even 100 epochs, I observed train loss decreasing but test accuracy being extremely low. I tried doing the same with the dgcnn classification and pointnet++ classification provided here, and I observed the same behavior. However, all these seem to work fine on ModelNet10. Is there some change that needs to be made before training on ModelNet40? 

Thanks!

**Comment by schakrab** [comment]
nevermind I have it working now
